cli/medperf/config.py

```diff
@@ -53,3 +54,18 @@
 statistics_timeout = None
 infer_timeout = None
 evaluate_timeout = None
+
+configurable_parameters = [
```

Comment:
This is fine, although it would be nice to think of a way of DRYing this up. If we ever decide to change the configurable parameters, we have to do it in two different places, which might be error-prone in the future.

Reply:
Exactly. I will create an issue for this to keep in mind, and add a comment as well.